samples/subsys/task_wdt/boards/nucleo_f091rc.overlay

```diff
@@ -0,0 +1,9 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * Copyright (c) 2022 STMicroelectronics
+ */
+
+&clk_lsi {
+	clock-frequency = <DT_FREQ_K(40)>;
+};
```

Comment:
LSI clock freq is not wdt specific.
Should it be put in board.dts ? If not a comment is required to explain this choice

Reply:
with comment
```
 /* 
 * Due to the LSI RC oscillator characteristics (defined by the datasheet),
 * the typical value set in the soc or board dts might be too low
 * for the IWWDG timeout : define the max value for this usecase.
 */
```